### ISSUE TITLE
Not working on macOS Monterey

### ISSUE THREAD

**Issue by ea7kir** [analogdevicesinc/libiio#865]
Although the package installs (if we ignore signing), it's not working on Monterey.

**Comment by mhennerich** [comment]
Not sure you are aware of this -
You can switch your Pluto to use USB CDC-NCM, which can replace the RNDIS and works out of the box on macOS.

https://wiki.analog.com/university/tools/pluto/users/customizing#changing_the_usb_ethernet_compatibility_mode

**Comment by tfcollins** [comment]
Installing from the pkg will put everything in the Frameworks folder. So to invoke iio_info would require a command like:
```bash
> /Library/Frameworks/iio.framework/Tools/iio_info -S
Library version: 0.24 (git tag: c4498c2)
Compiled with backends: xml ip usb
Available contexts:
        0: 192.168.86.231 (ad9361-phy,xadc,cf-ad9361-dds-core-lpc,cf-ad9361-lpc) [ip:analog.local.]
```

**Comment by tfcollins** [comment]
You will need to have zstd installed to use the pkg which can be installed through brew:
```
brew install zstd
```

**Comment by ea7kir** [comment]
Thanks, but I use this iMac for so many other things.   Could it mess up other apps I use? 

Regards,
Michael Naylor

**Comment by ea7kir** [comment]
PS.  I went ahead and installed zstd, rebooted, reinstalled, and I still get...

***@***.*** ~ % iio_info
zsh: command not found: iio_info

Does anything work these days?

Regards,
Michael Naylor

**Comment by pcercuei** [comment]
@tfcollins shouldn't zstd be installed in the .pkg if libiio depends on it?
Alternatively - we can disable the dependency in the CI for OSX, it is optional (for now).

**Comment by pcercuei** [comment]
@ea7kir you still need to run `/Library/Frameworks/iio.framework/Tools/iio_info` as this path isn't into the system's default search paths.

**Comment by ea7kir** [comment]
@pcercuei thanks. This does work, but I still can't access my ADALM-Pluto.  I now know this is because HoRNDIS is required, but is outdated and will not install on Monterey.  I don't think there's much I can do about this.

**Comment by mhennerich** [comment]
> @pcercuei thanks. This does work, but I still can't access my ADALM-Pluto. I now know this is because HoRNDIS is required, but is outdated and will not install on Monterey. I don't think there's much I can do about this.

Switch your unit to use CDC-NCM instead of RNDIS, I pasted the link in the posts above

**Comment by ea7kir** [comment]
@pcercuei I think I tried that, that was after I installed the F5EOE firmware which degrades to v0.31-dirty - ie pre revision D.  The Pluto is now back on the roof, ready for first time trials this evening.  I've created a document from several sources which I'm hoping will enable me to perform re-install/upgrades/patches remotely over the LAN, so if I do need to bring the Pluto back down to the shack I will try it again.  Thanks.

**Comment by pcercuei** [comment]
I think this is fixed now. Feel free to re-open if the problem still exists.

**Comment by ea7kir** [comment]
I did try that, but without success.  I’ll give it another go.

I think the 3rd party firmware I need for DATV, by F5EOE, overwrites most if not all of your firmware.  Whichever AD version I start with,  from v0.31 to v0.34, the logon screen always reverts to v0.31-4-g9ceb-dirty.  This could be why passwords don’t persist.

I recently added a patch to F5EOE, written by F5UII. After it destroyed itself I couldn’t reinstall your firmware.  It was then I discovered the mtd2 filesystem.  The only way to get back to sanity was to borrow next doors Windows laptop and face an OS I hadn’t used since XP.

I’m about to start over again, aiming for with the cleanest install possible, based on these - all of them.

- `boot.dfu` - DFU file for First Stage Boot Loader, and U-Boot
- `boot.frm` - Firmware file for First Stage Boot Loader, U-Boot and it's default environment
- `pluto.dfu` - DFU file for Pluto Firmware, this would include FPGA Bit File, Linux kernel (all drivers), and ram based file system
- `pluto.frm` - Firmware file for Pluto Firmware, this would include FPGA Bit File, Linux kernel (all drivers), and ram based file system
- `uboot-env.dfu` - DFU file which includes the default U-Boot environment

For example when would I wan to used `boot.dfu` and how?  Same quest for all the above.  And can these be copied/installed over ethernet without needing a Windows box?  And, in which order should they be installed?

Regards,
Michael Naylor
